docs/reporters.md

```diff
@@ -35,6 +35,17 @@ You see what reporters are available from the command line by running with `--li
 
 By default all these reports are written to stdout, but can be redirected to a file with [`-o` or `--out`](command-line.md#sending-output-to-a-file)
 
+<a id="multiple-reporters"></a>
```

Comment:
We usually use the heading ids for linking, but it might be a good idea to be inconsistent but have a stable link for cross-documentation linking.